app/apps/server/bridges/uploads.ts

```diff
@@ -7,6 +7,15 @@ import { FileUpload } from '../../../file-upload/server';
 import { determineFileType } from '../../lib/misc/determineFileType';
 import { AppServerOrchestrator } from '../orchestrator';
 
+
+const getUploadDetails = (details: IUploadDetails): Partial<IUploadDetails> => {
+	if (details.visitorToken) {
+		const { userId, ...result } = details;
+		return result;
+	}
+	const { visitorToken, ...result } = details;
+	return result;
```

Comment:
`visitorToken` was not deleted on previous code.
```suggestion
	return details;
```